packages/bsky/tests/views/threadgating.test.ts

```diff
@@ -440,6 +516,23 @@ describe('views with thread gating', () => {
       'multi rule reply allow (mention)',
     )
     await network.processAll()
+
+    const {
+      data: { thread: eveThread },
+    } = await agent.api.app.bsky.feed.getPostThread(
+      { uri: post.ref.uriStr },
+      {
+        headers: await network.serviceHeaders(
+          sc.dids.eve,
+          ids.AppBskyFeedGetPostThread,
+        ),
+      },
+    )
```

Comment:
General note on this file: these tests are getting kinda hard to understand. The block/follow relationships are not obvious and the order of these tests matters to some extent.

Optional, but if it's helpful to you, we could consider creating a new seed with users specific to each test case, [like this one.](https://github.com/bluesky-social/atproto/blob/threadgate/1-followers/packages/bsky/tests/seed/feed-hidden-replies.ts) That way we can have explicitly named users and write some tests for new rules that are easier to understand. Those tests can live in a separate file alongside this one.